metadata.c

```diff
@@ -422,6 +422,29 @@ int pv_metadata_rm_usermeta(const char *key)
 	return -1;
 }
 
+static void pv_unescape_data(char *buf, int size)
```

Comment:
I find this code a bit weird in general. Also, should it be somewhere in utils/ instead of here?

Reply:
str.h I'd say :)